diesel/src/query_dsl/limit_dsl.rs

```diff
@@ -6,6 +6,35 @@ use query_source::Table;
 pub trait LimitDsl {
     type Output;
 
+/// ```rust
+/// # #[macro_use] extern crate diesel;
+/// # include!("../doctest_setup.rs");
+/// # use schema::users;
+/// #
+/// # fn main() {
+/// #   use users::dsl::*;
+/// #   let connection = establish_connection();
+/// #   diesel::delete(users).execute(&connection).unwrap();
+/// #
+/// let new_users = vec![
```

Comment:
We don't typically show this setup. Can we instead do 2 selects, one with limit and one without, to show the difference that way?